include/zephyr/drivers/clock_control/stm32_clock_control.h

```diff
@@ -163,6 +163,8 @@
 #define STM32_CPUCLK_SRC_IC1	1
 #endif
 #endif /* cpusw clk source is rcc */
+
+#define STM32_TIMG_PRESCALER	DT_PROP_OR(DT_NODELABEL(rcc), timg_prescaler, 1)
```

Comment:
I think that it would be better to set the default value in the binding rather than here.

Reply:
I mean in the dts/bindings/clock/st,stm32n6-rcc.yaml file.
It's better to add `default: 1` here where the property is described rather than in the code where it is harder to find.

Reply:
Ok, crystal clear, thanks.